### PR TITLE
feat: added cobra cli pkg

### PR DESCRIPTION
**Pull request by swelf19**
Added cobra cli pkg with subcommand 
1) start -  starts the main relayer app
2) version - return version (git tag) and commit of the relayer build

### CODE REVIEW

File: cmd/neutron_query_relayer/cmd/start.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
```
Comment (pr0n00gler) [inline cmd/neutron_query_relayer/cmd/start.go:2]: Huh?

File: cmd/neutron_query_relayer/cmd/start.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	"sync"
+	"syscall"
+
+	nlogger "github.com/neutron-org/neutron-logger"
+	"github.com/neutron-org/neutron-query-relayer/internal/app"
+	"github.com/neutron-org/neutron-query-relayer/internal/config"
+	neutrontypes "github.com/neutron-org/neutron/x/interchainqueries/types"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
```
Comment (pr0n00gler) [inline cmd/neutron_query_relayer/cmd/start.go:22]: `goimports -w --local github.com/neutron-org .`

File: cmd/neutron_query_relayer/cmd/root.go [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "neutron_query_relayer",
+	Short: "Neutron Interchain Query Relayer",
```
Comment (pr0n00gler) [inline cmd/neutron_query_relayer/cmd/root.go:12]: Neutron Interchain Quer**ies** Relayer

File: cmd/neutron_query_relayer/cmd/root.go [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "neutron_query_relayer",
```
Comment (pr0n00gler) [inline cmd/neutron_query_relayer/cmd/root.go:11]: neutron_quer**ies**_relayer

File: .gitignore [hunk 5]
```diff
@@ -27,7 +27,6 @@ keys
 
 cmd/neutron_query_relayer/neutron_query_relayer
 storage
-neutron_query_relayer
```
Comment (oopcode) [inline .gitignore:30]: why?

File: cmd/neutron_query_relayer/cmd/root.go [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func init() {
+	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
```
Comment (oopcode) [inline cmd/neutron_query_relayer/cmd/root.go:25]: huh?

**Review by pr0n00gler — CHANGES_REQUESTED** [review]

**Review by oopcode — APPROVED** [review]
tested, everything seems to work

File: .gitignore (hunk 5 above)
Comment (NeverHappened) [reply .gitignore:30]: ah, he moved build artifact to build/ folder

File: cmd/neutron_query_relayer/cmd/root.go (hunk 3 above)
Comment (NeverHappened) [reply cmd/neutron_query_relayer/cmd/root.go]: Why? it's build as a binary named `neutron_query_relayer`

**Review by oopcode — DISMISSED** [review]

File: cmd/neutron_query_relayer/cmd/root.go (hunk 3 above)
Comment (pr0n00gler) [reply cmd/neutron_query_relayer/cmd/root.go]: If it requires a lot of changes, it's better to do it later of course

**Review by NeverHappened — CHANGES_REQUESTED** [review]
When I run it from integration tests (or neutron `make start-cosmopark`), I get an error:
```
Start relayer
./run.sh: line 8: /bin/neutron_query_relayer: cannot execute binary file: Exec format error
```

**Review by NeverHappened — APPROVED** [review]

**Review by sotnikov-s — APPROVED** [review]

**Pull request merged**